app/views/layouts/_header.html.erb

```diff
@@ -20,8 +20,10 @@
                                  service_url: service_link,
                                  navigation_items: NavigationItems.for_api_docs(controller))) %>
 <% else %>
+  <% components_url = '/rails/components' if request.path.match(/^\/rails\/components/) %>
+  <% components_name = 'ActionView::Component Previews' if request.path.match(/^\/rails\/components/) %>
   <%= render(HeaderComponent.new(classes: "app-header--#{HostingEnvironment.environment_name}",
-                                 service_name: service_name,
-                                 service_url: service_link,
+                                 service_name: components_name || try(:service_name),
+                                 service_url: components_url || try(:service_url),
```

Comment:
I wonder if there's a way to do this without it changing our production code. Could we somehow have a separate layout without worrying about getting out of sync with the proper frontend?

Reply:
I'd rather not to introduce a separate layout... Current modifications are not too bad, I think, anything more and we'll have to add a separate layout. What do you think?

Reply:
Let's live with it in here rather than hold back this PR